components/p3a/metric_names.h

```diff
@@ -84,7 +84,6 @@ inline constexpr auto kCollectedTypicalHistograms =
     {"Brave.Importer.ImporterSource.2", {}},
     {"Brave.NTP.CustomizeUsageStatus.2", {}},
     {"Brave.NTP.DefaultPage", {}},
-    {"Brave.NTP.NewTabsCreated.3", {}},
```

Comment:
Was this intentionally removed? It looks like the `kNewTabsCreated` metric is still being sent 🤔

Reply:
Yes, we're deprecating this metric. However, I plan to temporarily leave the old code in, just in case there's a stakeholder that wants to restore this metric in the near future.